pyupgrade.py

```diff
@@ -852,26 +852,56 @@ def _fix_super(contents_text):
     return tokens_to_src(tokens)
 
 
-class FindNewStyleClasses(ast.NodeVisitor):
+class SixVisitor(ast.NodeVisitor):
```

Comment:
instead of inheritance, let's combine `SixVisitor` and `FindsClasses` (I was actually going to do that as a followup anyway).  This will also improve performance of pyupgrade by ~15% as well :tada:

Reply:
I'll have you know this is enterprise-grade inheritance

Reply:
What about FindSixUsage?

Reply:
yeah, combine all 3